nut/server.sls

```diff
@@ -3,7 +3,7 @@
 
 {% from "nut/map.jinja" import nut with context %}
 {% set ups_config = nut.server.ups if nut.server.ups is defined else {} %}
-{% set upsd_config = nut.server.upsd.config if nut.server.upsd.config is defined else {} %}
+{% set upsd_config = nut.server.upsd.upsd if nut.server.upsd.upsd is defined else {} %}
```

Comment:
@martintamare, you're right it wrong, but it should actually be

```yaml
{% set upsd_config = nut.server.upsd if nut.server.upsd is defined else {} %}
```
to match the logic of `ups_config`. `upsd_users` is the 'special case' here.